### ISSUE TITLE
E3002: Additional properties are not allowed ('Tags' was unexpected) on AWS::SES::ConfigurationSet

### ISSUE THREAD

**Issue by mh35** [aws-cloudformation/cfn-lint#4203]
### CloudFormation Lint Version

1.38.0

### What operating system are you using?

macOS

### Describe the bug

Recently, AWS::SES::ConfigurationSet started to support Tags attribute, but error occurs.

### Expected behavior

AWS::SES::ConfigurationSet supports Tags attribute, and the Tags attribute should be validated as other resources.

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  A:
    Type: AWS::SES::ConfigurationSet
    Properties:
      ReputationOptions:
        ReputationMetricsEnabled: true
      SendingOptions:
        SendingEnabled: true
      SuppressionOptions:
        SuppressedReasons:
          - COMPLAINT
          - BOUNCE
      Tags:
        - Key: Name
          Value: !Sub ${AWS::StackName}-SesConfigSet
```
